src/shell/shell.cpp

```diff
@@ -1258,6 +1260,31 @@ void SHELL_Init() {
 	        " Volume in drive %c is %s\n"
 	        " Volume Serial Number is %04X-%04X\n"
 	        "\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP",
+	        "Moves files and renames files and directories\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP_LONG",
+	        "Usage:\n"
+	        "To move one or more files:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]filename1[,...] destination[reset]\n"
```

Comment:
`[color=cyan][drive:][path]filename1[,...]` -> `[color=cyan]FILE[reset] [[color=cyan]FILE[reset] ...]` - for uniformity with other commands
`destination` -> `[color=cyan]DESTINATION[reset]` - compare with COPY command help string

You can explain in the description that a file can contain a path.

Reply:
Consistency is important.